lib/presentation/screen/license_agreement_screen.dart

```diff
@@ -0,0 +1,54 @@
+import 'package:flutter/material.dart';
+import 'package:star_book/presentation/shared/elevated_buttons.dart';
+import 'package:star_book/theme/styling/theme_color_style.dart';
+
+class LicenseAgreementScreen extends StatelessWidget {
+  const LicenseAgreementScreen({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    final double screenHeight = MediaQuery.of(context).size.height;
+    return Scaffold(
+      body: SafeArea(
+        child: Padding(
+          padding: const EdgeInsets.symmetric(horizontal: 24),
+          child: Column(
```

Comment:
screen is scrollable with fixed bottom button

Reply:
Okay